docs/en/reference/types.rst

```diff
@@ -463,7 +465,7 @@ Types that map to objects such as POPOs.
 object
 ^^^^^^
 
-Maps and converts object data based on PHP serialization.
+Maps and converts object data using PHP's ``serialize()`` and ``unserialize()``.
```

Comment:
Implementation details, the current version is fine.

Reply:
Let's take a look at the current version:

> Maps and converts object data based on PHP **serialization**. If you need to store an exact representation of your object data, you should consider using this type as it uses **serialization** to represent an exact copy of your object as string in the database. Values retrieved from the database are always converted to PHP's object type using **deserialization** or null if no data is present.

The same pseudo-information ("serialization") is repeated over and over; but which form of serialization is hidden from the user. Why?
So ultimately, I'd suggest to add a link to https://www.php.net/manual/function.serialize.php - then half of this paragraph can be deleted.

Reply:
>  which form of serialization is hidden from the user.

:thinking: it says "PHP serialization"… are there several forms of PHP serialization?

Reply:
I don't know.
But if there's just one anyway, we can call it by its name; instead of leaving users uncertain.

Reply:
I'm really not convinced.

Reply:
It's like explaining how the motor of a car works while the user just wants to drive a car. If there are other places where the serialization got mentioned, then it's because we didn't know better about mentioning details.

Reply:
Guys, I don't know what to say here. The real problem on that page is the horrible [Mapping Matrix](https://www.doctrine-project.org/projects/doctrine-dbal/en/3.8/reference/types.html#mapping-matrix), but instead of discussing how it can be improved (i.e. removed), we keep arguing for three months if "PHP serialization" is a better term than `serialize()`?!

As I said, all I did was to **copy** it over from https://github.com/doctrine/orm/blame/3.0.x/docs/en/reference/basic-mapping.rst#L257, where it has been sitting for some 14 years - and nobody of you cared.

> are there several forms of PHP serialization?

Searching for "serialize" on https://www.php.net/ gives me 28 hits.

So if everybody agrees with @derrabus that the current version is "perfectly fine", you can just close here...

Reply:
> The real problem on that page is the horrible [Mapping Matrix](https://www.doctrine-project.org/projects/doctrine-dbal/en/3.8/reference/types.html#mapping-matrix),

But that's not what this PR does, is it.

> but instead of discussing how it can be improved (i.e. removed), we keep arguing for three months if "PHP serialization" is a better term than `serialize()`?!

Right, we should've closed this PR sooner.